test/integration/targets/efs_facts/playbooks/roles/efs_facts/tasks/main.yml

```diff
@@ -99,12 +100,45 @@
           - efs_result.ansible_facts.efs[0].encrypted == false
           - efs_result.ansible_facts.efs[0].life_cycle_state == "available"
           - efs_result.ansible_facts.efs[0].performance_mode == "generalPurpose"
+          - efs_result.ansible_facts.efs[0].throughput_mode  == "bursting"
           - efs_result.ansible_facts.efs[0].mount_targets[0].security_groups[0] == vpc_default_sg_id
           - efs_result.ansible_facts.efs[0].mount_targets[1].security_groups[0] == vpc_default_sg_id
 
     - assert:
         that: "{{efs_result_assertions}}"
 
+    # ============================================================
+    - name: Update Efs to use provisioned throughput_mode
+      efs:
+        <<: *aws_connection_info
+        state: present
+        name: "{{ resource_prefix }}-test-efs"
+        tags:
+            Name: "{{ resource_prefix }}-test-tag"
+            Purpose: file-storage
+        targets:
+            - subnet_id: "{{testing_subnet_a.subnet.id}}"
+            - subnet_id: "{{testing_subnet_b.subnet.id}}"
+        throughput_mode: 'provisioned'
+        provisioned_throughput_in_mibps: 5.0
+
+    - pause: seconds=3
+    # ============================================================
```

Comment:
Would be good to also test that: 
* running in the same values returns `changed: False`
* updating `provisioned_throughput_in_mibps` correctly changes the configuration and that the `efs` module returns this configuration (it's good to test efs_facts too, but the efs module should also return the relevant info)